aemedge/blocks/channel-lookup/channel-lookup.js

```diff
@@ -0,0 +1,114 @@
+import { createTag } from '../../scripts/utils.js';
+
+const ZIPCODE_ENDPOINT = 'https://p-geo.movetv.com/geo';
+const DEFAULT_ZIPCODE = '90020';
+const ZIPCODE_KEY = 'user_zip';
+let gqlResponse = '';
+
+export async function getZipcode() {
```

Comment:
since we are using this function in both zipcode block and here, it may makes sense to put it into utils.js and refactor both zipcode.js and here to use the method from utils

Reply:
done, please review